lib/bgsite_official/categories/tag.ex

```diff
@@ -17,4 +17,20 @@ defmodule BgsiteOfficial.Categories.Tag do
     |> cast(attrs, [:title, :banner, :description, :likes])
     |> validate_required([:title, :banner, :description, :likes])
   end
+
+  def changeset_update_projects(%Tag{} = tags, attrs) do
```

Comment:
I think the name should not include `projects`, but something like `changeset_update_tags_for_websites`.